Scripts/Tracking/Follow/Modifier/ModifyTransform.cs

```diff
@@ -12,44 +12,29 @@ public class ModifyTransform : FollowModifier
         /// </summary>
         /// <param name="source">The source <see cref="Transform"/> to modify.</param>
         /// <param name="target">The target <see cref="Transform"/> to utilize in the modification.</param>
-        public override void UpdatePosition(Transform source, Transform target)
+        protected override void DoUpdatePosition(Transform source, Transform target)
         {
-            CachedSource = source;
-            CachedTarget = target;
-            if (source != null && target != null)
-            {
-                source.position = target.position;
-            }
+            source.position = target.position;
         }
 
         /// <summary>
         /// Sets the source <see cref="Transform.rotation"/> to the target <see cref="Transform.rotation"/>.
         /// </summary>
         /// <param name="source">The source <see cref="Transform"/> to modify.</param>
         /// <param name="target">The target <see cref="Transform"/> to utilize in the modification.</param>
-        public override void UpdateRotation(Transform source, Transform target)
+        protected override void DoUpdateRotation(Transform source, Transform target)
         {
-            CachedSource = source;
-            CachedTarget = target;
-            if (source != null && target != null)
-            {
-                source.rotation = target.rotation;
-            }
+            source.rotation = target.rotation;
         }
 
         /// <summary>
         /// Sets the source <see cref="Transform.localScale"/> to the target <see cref="Transform.localScale"/>.
         /// </summary>
         /// <param name="source">The source <see cref="Transform"/> to modify.</param>
         /// <param name="target">The target <see cref="Transform"/> to utilize in the modification.</param>

```

Comment:
This XML docs block should just be `<inheritdoc />`.

Reply:
The docs are different for this method